app/docs/charts.stories.tsx

```diff
@@ -91,7 +91,11 @@ const ColumnsStory = {
         <InteractiveFiltersChartProvider chartConfigKey={chartConfig.key}>
           <Observer>
             <ColumnChart
-              limits={{ axisDimension: undefined, limits: [] }}
+              limits={{
+                axisDimension: undefined,
+                limitMeasure: undefined,
+                limits: [],
+              }}
```

Comment:
Seems like limits is optional, so we could just remove it ?

Reply:
Ah, they are required in the chart types 🥹 

![code](https://github.com/user-attachments/assets/996309f9-17d0-427c-aa4e-a2f2f1a5fab5)

<img width="978" alt="Screenshot 2025-05-13 at 10 35 54" src="https://github.com/user-attachments/assets/42ecccf2-34c8-4a3c-a8b3-c61f2c517c0a" />

Reply:
OK, they could be made optional on the chartType right ? It's not a big problem, but it's a bit annoying :sweat\_smile: No problem if you have other things to do.

Reply:
It sounds like we still would need to work on the limits to support more types in the near future, I'll make a note to myself 👍